panther_gazebo/launch/simulate_robot.launch.py

```diff
@@ -80,6 +104,17 @@ def generate_launch_description():
         choices=["True", "true", "False", "false"],
     )
 
+    declare_wheel_type_arg = DeclareLaunchArgument(
+        "wheel_type",
+        default_value=PythonExpression(["'WH01' if '", robot_model, "' == 'panther' else 'WH05'"]),
```

Comment:
```suggestion
        default_value=PythonExpression(["'WH05' if '", robot_model, "' == 'lynx' else 'WH01'"]),
```
I would assume default value for panther